### DISCUSSION TITLE
Why even build for Swift 5.3 and 5.4 when my package says it requires Swift 5.5?

### DISCUSSION THREAD

**Discussion by kevinrenskers** [SwiftPackageIndex/SwiftPackageIndex-Server#1666] [Ideas]
If you look at https://swiftpackageindex.com/loopwerk/Saga/builds for example, you'll see a bunch of crosses for Swift 5.3 and 5.4. I think this comes across a bit negative as the package states it requires Swift 5.5, but okay. Not a huge deal.

I think the bigger question is: why waste resources actually running these builds, when you can determine beforehand it's not going to work anyway?

**Comment by daveverwer** [comment]
Thanks for the suggestion Kevin! This comes up reasonably regularly. See #727 for our original comments on it. 

We do now have the tools version stored, which we could use to skip builds, but they really do fail very quickly and so I'm still not sure the additional complexity is worthwhile.

**Reply by kevinrenskers** [reply]
Fair enough!
